google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/application/AppEngineApplicationCreateDialog.java

```diff
@@ -147,8 +147,8 @@ private void trackApplicationCreateFailure() {
         .ping();
   }
 
-  private void setStatusMessageAsync(final String message, final boolean isError) {
```

Comment:
does errorProne really complain about this? even though its always true (since we don't have other use cases yet), it feels weird to be hardcoding it to true in this method. Not a big deal though